sdk/core/Cargo.toml

```diff
@@ -59,6 +59,7 @@ enable_reqwest_gzip = ["reqwest/gzip"]
 enable_reqwest_rustls = ["reqwest/rustls-tls"]
 hmac_rust = ["dep:sha2", "dep:hmac"]
 hmac_openssl = ["dep:openssl"]
+test = []
```

Comment:
Can this be named something more self-describing?

Reply:
Yes, open to suggestions. May be `unit_test`?

Reply:
I renamed it `unit_test`. Were you suggesting something even more specific like `noop_client`?

Reply:
Renamed to `noop_client`.